.github/workflows/lint-test.yaml

```diff
@@ -75,7 +75,7 @@ jobs:
           node-version-file: .node-version
       - uses: pnpm/action-setup@v2
         with:
-          version: 7.14.2
+          version: 8.2.0
```

Comment:
Updating just one project to a different pnpm version will be a pain, particularly since it has a different lock file format. Let's land this first, I update everything to this new pnpm version: https://github.com/aptos-labs/aptos-core/pull/7890.